packages/rxjs/test/index.test.js

```diff
@@ -0,0 +1,68 @@
+const { TestScheduler } = require('rxjs/testing');
+const { from, of } = require('rxjs');
+const { delay, concatMap } = require('rxjs/operators')
+const { deepEqual } = require('assert');
+
+const { getNames, throttleTimeForThreeMs } = require('../custom-operator');
+
+describe('observable stream', ()=> {
+  let testScheduler;
+  const sourceValues = [
+    {
+      name: 'mocha'
+    }
+    ,{
+      name: 'vanilla'
+    }
+    ,{
+      name: 'caramel'
+    }
+  ]
+
+  beforeEach(()=> {
+    testScheduler = new TestScheduler((actual, expected) => {
+      deepEqual(actual, expected);
+    });
+  })
+
+  it('should follow marble diagrams with basic operators', ()=> {
+    testScheduler.run(({ cold, expectObservable, expectSubscriptions }) => {
+      const targetStream =  cold('01(2|)', sourceValues);
+      const subs =               '^-!';
```

Comment:
Odd formatting.

Reply:
This formatting is intended like [RxJs internal spec](https://github.com/ReactiveX/rxjs/blob/f35e622c24445ac9d5631538e2abd748e9c3bbb5/spec/operators/mergeMap-spec.ts#L24), but it would be redundant in this case because my test is very simple and easy. Thank you!